app/controllers/editorial/submissions_controller.rb

```diff
@@ -10,7 +10,7 @@ class Editorial::SubmissionsController < Editorial::SectionsController
 
 
     def index
-      @submissions = scope.open.for(current_user)
+      @submissions = scope.open
```

Comment:
We didn't have sorting previously though did we? Or was that sort hidden inside `for` scope / method....? (looking it up)